src/web/app/pages-instructor/instructor-home-page/__snapshots__/instructor-home-page.component.spec.ts.snap

```diff
@@ -0,0 +1,2741 @@
+                  aria-haspopup="true"
+                  class="btn btn-secondary btn-sm dropdown-toggle"
+                  ngbdropdowntoggle=""
+                >
+                   Course 
+                </button>
+                <div
+                  class="dropdown-menu"
+                  ngbdropdownmenu=""
+                  x-placement="bottom-left"
+                >
+                  <a
+                    class="btn btn-light btn-sm dropdown-item"
+                    ng-reflect-ngb-tooltip="Archive the course so that it "
+                    ngbtooltip="Archive the course so that it will not be shown in the home page any more (you can still access it from the 'Courses' tab)"
+                  >
+                     Archive 
+                  </a>
+                  <a
+                    class="btn btn-light btn-sm dropdown-item"
+                    href="/web/instructor/courses/edit?courseid=CS3281"
+                    ng-reflect-ngb-tooltip="Edit course information and in"
+                    ng-reflect-query-params="[object Object]"
+                    ng-reflect-router-link="/web/instructor/courses/edit"
+                    ngbtooltip="Edit course information and instructor list"
+                    routerlink="/web/instructor/courses/edit"
+                  >
+                     View / Edit 
+                  </a>
+                  
+                  
+                  <a
+                    class="btn btn-light btn-sm dropdown-item"
+                    ng-reflect-ngb-tooltip="Delete the course and its corr"
+                    ngbtooltip="Delete the course and its corresponding students and sessions"
+                  >
+                     Delete 
+                  </a>
+                  
+                </div>
+              </span>
+              
+              <i
+                class="fas fa-chevron-down"
+              />
+              
+            </div>
+          </div>
+        </div>
+      </div>
+      
+      
+      
+      
+    </div>
+  </div>
+</tm-instructor-home-page>
+`;
+
+exports[`InstructorHomePageComponent should snap with one course with unpopulated feedback sessions 1`] = `
+<tm-instructor-home-page
+  SessionsTableColumn={[Function Object]}
+  SessionsTableHeaderColorScheme={[Function Object]}
+  SortBy={[Function Object]}
+  courseService={[Function CourseService]}
+  courseTabModels={[Function Array]}
+  feedbackQuestionsService={[Function FeedbackQuestionsService]}
+  feedbackSessionsService={[Function FeedbackSessionsService]}
+  httpRequestService={[Function HttpRequestService]}
+  instructorCoursesSortBy={[Function Number]}
+  navigationService={[Function NavigationService]}
+  ngbModal={[Function NgbModal]}
+  route={[Function ActivatedRoute]}
+  router={[Function Router]}
+  statusMessageService={[Function StatusMessageService]}
+  studentSearchkey=""
+  timezoneService={[Function TimezoneService]}
+  user=""
+>
+  <h1>
+    Home
+  <
```

Comment:
This session table renders the header while not having any feedback session in it. Not exactly wrong, but the more ideal way is to display something else e.g. "There are no feedback sessions in this course."